gateway/include/webConfigurer.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <WebConfig.h>
+
+/// @brief Creates a web server and configures the data to display (on the web server), to save and to fetch (on SPIFFS).
+class WebConfigurer {
+ public:
+  WebConfigurer() = default;
+
+  /// @brief Configures WebConfig with defined parameters
+  void configure();
+
+  /// @brief Resets all WebConfig parameters
+  void reset();
+
+  /// @brief Listen on port 80
+  void serverListen();
+
+  /// @brief Close the web server
+  void serverClose();
+
+  /// @brief Handle client connection (must be in a for loop)
+  void handleClient();
+
+  // GETTERS for each configuration parameter
+  
+  /// @brief Gets Wifi Ssid to initiate connection
+  /// @return the wifi Ssid string
+  String getWifiSsid();
```

Comment:
Return const String by reference.  
Same for those below.  
You will also have to change .cpp file.

```suggestion
  const String& getWifiSsid();
```